packages/client/helpers/functional-test/run-tests.ts

```diff
@@ -154,7 +154,12 @@ async function main(): Promise<number | void> {
     )
 
     if (unknownAdapterProviders.length > 0) {
-      throw new Error(`Unknown adapter providers: ${unknownAdapterProviders.join(', ')}`)
+      const allAdaptersStr = Array.from(allAdapterProviders)
```

Comment:
Not strictly related to this PR, it just took me a while to figure out that proper `--adapter` argument is `js_d1` so I think it would be quicker if we list all supported options when we see unknown value.